bugsnag-plugin-android-ndk/src/main/jni/bugsnag_ndk.h

```diff
@@ -28,15 +28,15 @@ typedef struct {
      * Preferred unwinding style
      */
     bsg_unwinder unwind_style;
-    bsg_report_header report_header;
+    bsg_event_header report_header;
```

Comment:
As we're still calling the file a "report", this struct should probably still be bsg_report_header. Or else we should rename the variable and several of the methods.

Also, although this is a previous omission, can we give this a comment to document it?

Reply:
Renamed the struct to `bsg_report_header` and added a comment to this field.